app/editor/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "lib": [
       "dom",
       "dom.iterable",
-      "esnext"
+      "esnext",
+      "ES2021"
```

Comment:
This was an attempt to fix linting error where string function does not exist.  It however, didn't seem to change anything in the IDE.